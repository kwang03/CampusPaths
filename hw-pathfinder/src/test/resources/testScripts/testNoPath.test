#Tests Dijkstra's when there is no path between the two requested nodes

CreateGraph myGraph

AddNode myGraph A

AddNode myGraph B

AddNode myGraph C

AddEdge myGraph A B 10.0

FindPath myGraph A C