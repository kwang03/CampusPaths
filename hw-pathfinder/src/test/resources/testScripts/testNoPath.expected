#Tests Dijkstra's when there is no path between the two requested nodes

created graph myGraph

added node A to myGraph

added node B to myGraph

added node C to myGraph

added edge 10.000 from A to B in myGraph

path from A to C:
no path found